src/lib/Behat/Component/Table/TableRow.php

```diff
@@ -83,7 +83,7 @@ protected function specifyLocators(): array
             new VisibleCSSLocator('link', 'a'),
             new VisibleCSSLocator('checkbox', 'input[type=checkbox]'),
             new VisibleCSSLocator('assign', '[data-bs-original-title="Assign content"],[data-bs-original-title="Assign to Users/Groups"]'),
-            new VisibleCSSLocator('edit', '.ibexa-icon--edit,[data-bs-original-title="Edit"]'),
+            new VisibleCSSLocator('edit', '.ibexa-icon--edit,[data-bs-original-title="Edit"], [href*="/update"]'),
```

Comment:
Could you provide brief explanation why this change is done? 🙂

Reply:
I'm using function edit from TableRow class. Edit function has locator for specific button(edit) in the table.

Reply:
Have you seen https://github.com/ibexa/admin-ui/pull/439? 🙂 \cc @katarzynazawada